docs/metrics.md

```diff
@@ -4,6 +4,9 @@ For the metrics capability, Firecracker uses a single Metrics system.
 This system can be configured by sending a `PUT` API Request to the
```

Comment:
We should mention that the metrics can be configured **either** through API or CLI. Also we can have a better description in the swagger file [here](https://github.com/firecracker-microvm/firecracker/blob/d2d1bf5dcd41c9eca16be055a724805cfe5b3e4f/src/api_server/swagger/firecracker.yaml#L369).

It would also be nice to have a test for this behaviour.

Reply:
How about this for the 1st para:

> For the metrics capability, Firecracker uses a single Metrics system. This
system can be configured either by: a) sending a `PUT` API Request to the
`/metrics` path: or b) using the `--metrics-path` CLI option.

Reply:
Yup, looks good. I was thinking to add a test where you configure the metrics through CLI and then try to re-configure through API and expect a failure.